multistream.js

```diff
@@ -2,7 +2,7 @@
 
 const metadata = Symbol.for('pino.metadata')
 
-const levels = {
+var levels = {
```

Comment:
Why the revert of `const` to `var`?

Reply:
@jsumners This was a global override, which in second-hand isn't best practice. I'll update the PR with a reference to level that is separated into `multistream` only

Reply:
I've updated the closure, when time found please do have a look.